### DISCUSSION TITLE
Option to download order pdfs as one combined pdf instead of zip

### DISCUSSION THREAD

**Discussion by WHITE-developer** [craftcms/commerce#2425] [Ideas] [answered]
**What are you trying to do?**
Downloading multiple order pdfs which now generates an zip file

**What's your proposed solution?**
Add an option to instead get a combined pdf

**Additional context**
This would be handy for printing a package slip. One pdf so only hit the print button once instead of opening multiple pdfs from the zip file

**Comment by lukeholder** [comment]
Thanks for the request, added it as a future enhancement.

**Comment by WHITE-developer** [comment]
@lukeholder Just curious, is it in the pipeline somewhere? A lot of our customers would love to have it.

**Comment by lukeholder** [comment]
@WHITE-developer This will be in the next release: #2179

**Comment by lukeholder** [comment]
Apologies, due to the feature needing UI translations we need to delay this to 3.4 but the feature is complete here: https://github.com/craftcms/commerce/pull/2190

**Comment by lukeholder** [comment, marked as answer]
This was completed in Commerce 3.4.0

**Comment by WHITE-developer** [comment]
@lukeholder Could there be a setting to set the default way of downloading (zip or combined), some of our clients are asking if we could set the combined one as default as this would take them less clicks/time
